Polar/inputs-preprocess/subglacial-lakes-2013-2017.py

```diff
@@ -0,0 +1,45 @@
+import xarray as xr
```

Comment:
This script leads in error: 

`15:14 $ python inputs-preprocess/subglacial-lakes-2013-2017.py`

```
Traceback (most recent call last):
  File "/home/conda/deepesdl/envs/cube-gen/lib/python3.9/site-packages/xarray/backends/file_manager.py", line 201, in _acquire_with_cache_info
    file = self._cache[self._key]
  File "/home/conda/deepesdl/envs/cube-gen/lib/python3.9/site-packages/xarray/backends/lru_cache.py", line 55, in __getitem__
    value = self._cache[key]
KeyError: [<function open at 0x7fdd52c99820>, ('/home/alicebalfanz/data/polar/source/repeat_subglacial_lake_drainage_and_filling_thwaites_glacier/Thw70_2013_mask.tif',), 'r', (('sharing', False),)]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "rasterio/_base.pyx", line 307, in rasterio._base.DatasetBase.__init__
  File "rasterio/_base.pyx", line 218, in rasterio._base.open_dataset
  File "rasterio/_err.pyx", line 221, in rasterio._err.exc_wrap_pointer
rasterio._err.CPLE_OpenFailedError: /home/alicebalfanz/data/polar/source/repeat_subglacial_lake_drainage_and_filling_thwaites_glacier/Thw70_2013_mask.tif: No such file or directory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/alicebalfanz/cube-gen/Polar/inputs-preprocess/subglacial-lakes-2013-2017.py", line 35, in <module>
    da_2013 = to_xarray(rioxarray.open_rasterio(lake_path_2013),lake,2013)
  File "/home/conda/deepesdl/envs/cube-gen/lib/python3.9/site-packages/rioxarray/_io.py", line 916, in open_rasterio
    riods = manager.acquire()
  File "/home/conda/deepesdl/envs/cube-gen/lib/python3.9/site-packages/xarray/backends/file_manager.py", line 183, in acquire
    file, _ = self._acquire_with_cache_info(needs_lock)
  File "/home/conda/deepesdl/envs/cube-gen/lib/python3.9/site-packages/xarray/backends/file_manager.py", line 207, in _acquire_with_cache_info
    file = self._opener(*self._args, **kwargs)
  File "/home/conda/deepesdl/envs/cube-gen/lib/python3.9/site-packages/rasterio/env.py", line 444, in wrapper
    return f(*args, **kwds)
  File "/home/conda/deepesdl/envs/cube-gen/lib/python3.9/site-packages/rasterio/__init__.py", line 304, in open
    dataset = DatasetReader(path, driver=driver, sharing=sharing, **kwargs)
  File "rasterio/_base.pyx", line 309, in rasterio._base.DatasetBase.__init__
rasterio.errors.RasterioIOError: /home/alicebalfanz/data/polar/source/repeat_subglacial_lake_drainage_and_filling_thwaites_glacier/Thw70_2013_mask.tif: No such file or directory
```

Reply:
Seems like this data has been downloaded into the polar directory: 

![image](https://user-images.githubusercontent.com/23457217/219309293-a5ba41ab-ade6-4a5e-afb6-e2661426a595.png)

Reply:
Solved